keyboards/iris/keymaps/edvorakjp/keymap.c

```diff
@@ -0,0 +1,171 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```

Comment:
Heads up, the action_layer and eeconfig includes aren't actually needed.  

No need to change this, but just wanted to let you know.